Developer tools/Code editors and browsers.md

```diff
@@ -0,0 +1,46 @@
+## Code editors and IDE
+
+We all have our prefered way of writing code. It's something you get accoustomed to, and it almost becomes a second-nature to you.
+
+Over time you add a few shortcuts here and there. You discover some cool and useful plugins that will enhance your productivty.
+
+As time goes by you become well accustomed to your editor and it's a pain to switch to a new cool editor.
+
+And we don't expect you to.
+
+We don't care what you use to write your code. Unless it's on paper. That's a bit hard to maintain, plus it's hard to run automated checks on it 😄.
+
+Today you have a plethora of useful editors and IDEs you can develop your code on.
+
+We'll give you a list of editors/IDEs we are using, or that we used in the past (and they still occupy a special place in our hart).

```

Comment:
```suggestion
We'll give you a list of editors/IDEs we are using, or that we used in the past (and they still occupy a special place in our heart).

```
Or maybe even `hearts`, as a plural. Not 100% sure which would be better in this case. 😅

Reply:
I think hearts sounds better 👍🏼